### PR TITLE
Updated USA intern guide

### PR DESCRIPTION
**Pull request by aam1r**
Added some information about apartment hunting, bank/credit card, and
relocation. Also fixed some typos.

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -136,6 +146,8 @@ Have a second (or third) copy of all travel documents:
 Don't call it a job or say you are working! The J1 visa status is not meant for "working", but rather for "training" in the United States.
 Remember, you're just a poor little intern *training* in the United States!
 
+I prefer keeping all the above as a digital copy in my Dropbox/Google Docs so I have instant access to them just in case I lose anything.
```
Comment (sholiday) [inline README.markdown:149]: Ooh! Good tip. I do the same!

File: README.markdown [hunk 2]
```diff
@@ -168,13 +181,15 @@ Chase is OK there too.
 
 Wells Fargo is known for being super nice.
 
+Citibank has a Student account (no fees, and allows you to sign-up without an SSN). I signed up for an account while I was still in a hotel and didn't have a social security number yet.
```
Comment (sholiday) [inline README.markdown:184]: You shouldn't need a SSN to open a bank account at BoA, Chase, or Wells Fargo either,
But that stuff should be mentioned.

File: README.markdown (hunk 1 above)
Comment (aam1r) [inline README.markdown:149]: Documents in the cloud ftw.

File: README.markdown [hunk 3]
```diff
@@ -168,13 +181,15 @@ Chase is OK there too.
...
+
 ### <a name="CreditCards"></a> Credit Cards
 One thing that I and many others try to do right away is get a credit card.
 
 Getting credit in the US is very hard. Especially if you are under 21.
 
 Keep in mind that instead of Interac for debit cards, all American debit cards (also known as "check cards") are actually VISA/MasterCard cards.
-You can use them anywhere you would use a credit card except the money comes out of your account immediately.
+You can use them anywhere you would use a credit card except the money comes out of your account immediately. Citibank (see above) has all their checking debit cards to be "MasterCard debit cards".
```
Comment (sholiday) [inline README.markdown:192]: Not sure what you mean by that.

File: README.markdown (hunk 1 above)
Comment (axyjo) [inline README.markdown:149]: Me too!

File: README.markdown (hunk 2 above)
Comment (axyjo) [inline README.markdown:184]: Verified with Chase.

File: README.markdown (hunk 3 above)
Comment (axyjo) [inline README.markdown:192]: I think he just means that Citibank's cards are issued by MC rather than Visa.

Comment (aam1r) [inline README.markdown:192]: Uhh yeah, you're right -- it doesn't make sense. What I meant is that all Citibank debit cards have a Mastercard logo which means they can be used either a debit card or a credit card. You already mentioned that though so I guess I should take it out.

**Pull request merged**